mappings/net/minecraft/util/math/BlockBox.mapping

```diff
@@ -27,28 +27,34 @@ CLASS net/minecraft/class_3341 net/minecraft/util/math/MutableIntBoundingBox
 	METHOD method_14657 intersects (Lnet/minecraft/class_3341;)Z
 		ARG 1 other
 	METHOD method_14658 toNbt ()Lnet/minecraft/class_2495;
-	METHOD method_14659 getSize ()Lnet/minecraft/class_2382;
-	METHOD method_14660 getBlockCountX ()I
-	METHOD method_14661 translate (III)V
+	METHOD method_14659 getDimensions ()Lnet/minecraft/class_2382;
+	METHOD method_14660 getXLength ()I
+	METHOD method_14661 offset (III)V
 		ARG 1 dx
 		ARG 2 dy
 		ARG 3 dz
 	METHOD method_14662 contains (Lnet/minecraft/class_2382;)Z
 		ARG 1 vec
-	METHOD method_14663 getBlockCountY ()I
-	METHOD method_14664 getBlockCountZ ()I
+	METHOD method_14663 getYLength ()I
+	METHOD method_14664 getZLength ()I
```

Comment:
I think it should stay block count to avoid confusion between "length" and "dimensions". The `getDimensions` method returns max - min:

    public Vec3i getSize() {
        return new Vec3i(this.maxX - this.minX, this.maxY - this.minY, this.maxZ - this.minZ);
    }

while the `getBlockCount` methods return max - min + 1 (the number of blocks, so a box from (0, 0, 0) to (0, 0, 0) will have a block count of 1 since it contains the block at (0, 0, 0):

    public int getBlockCountX() {
        return this.maxX - this.minX + 1;
    }
    
    public int getBlockCountY() {
        return this.maxY - this.minY + 1;
    }
    
    public int getBlockCountZ() {
        return this.maxZ - this.minZ + 1;
    }

Reply:
Yeah, I think so. It was originally changed to `getX/Y/ZSize` to conform with the method in `Box` but now that the class has been renamed to `BlockBox` there is no longer any need to maintain parity.

Reply:
The names are _horribly_ unwieldy and clunky to read, though, so I welcome any, literally _any_ other suggestions.

Reply:
I reckon this can always be changed later if someone has a better idea.